### ISSUE TITLE
Document PhpStorm tips

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/documentation#7155]
* How to use it for Nextcloud (one project, apps are just sub dirs)
* How to configure coding style
  + Single line prop style ![20210928_175358](https://user-images.githubusercontent.com/1374172/135216315-1d355ad5-406e-4cfe-b89c-6ec51883a3bf.png)
* How to configure license headers
* How to run tests

**Comment by skjnldsv** [comment]
Still relevant @ChristophWurst ? 🙈

**Comment by ChristophWurst** [comment]
Yes but we can close it. There is also no documentation/help for other environments.
